File: monitor/v1/alert/read-example-warnings-apr01-apr30/read-example-warnings-apr01-apr30.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Converters;
using Twilio.Rest.Monitor.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var alerts = AlertResource.Read(
            endDate: MarshalConverter.DateTimeFromString("2015-04-30"),
            logLevel: "warning",
            startDate: MarshalConverter.DateTimeFromString("2015-04-01")
        );

        foreach(var record in alerts)
        {
           Console.WriteLine(record.Sid);
        }
    }
}

File: monitor/v1/alert/read-example-warnings-apr01-apr30/read-example-warnings-apr01-apr30.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

alerts = @client.monitor.alerts.list(
                                  end_date: Date.new(2015, 4, 30),
                                  log_level: 'warning',
                                  start_date: Date.new(2015, 4, 1)
                                )

alerts.each do |record|
  puts record.sid
end
